src/app/globals.css

```
@@ -163,6 +163,23 @@ body {
     color: var(--text-secondary);
   }
 
+  /* On a call: a live-broadcast chip that breathes */
+  .on-call {
+    @apply inline-flex items-center gap-1 rounded px-1.5 py-0.5 text-[11px] font-semibold tracking-wide uppercase;
+    background: color-mix(in srgb, var(--danger, #ef4444) 18%, transparent);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)